worlds/animal_well/options.py

```diff
@@ -71,6 +71,15 @@ class CandleChecks(Toggle):
     display_name = "Candle Checks"
 
 
+class Fruitsanity(Toggle):
+    """
+    Each fruit sends a check after being eaten.
```

Comment:
```suggestion
    All 115 fruits send a check after being eaten.
```
I like stating the number of checks added when having an option like this